cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -20,6 +20,18 @@
     - file_mode: 644
 {% endif %}
 
+{% if pillar.get('enable_cluster_monitoring', '').lower() == 'googleinfluxdb' %}
+/etc/kubernetes/addons/cluster-monitoring/googleinfluxdb:
+  file.recurse:
+    - source: salt://kube-addons/cluster-monitoring
```

Comment:
I am little bit confused here: you introduced a new mode called googleinfluxdb which enable both GCM and influxdb, right? If I understood it correctly, shouldn't the source here points to  cluster/addons/cluster-monitoring/googleinfluxdb which contains yaml files defining this mode?